Library/Formula/kbtin.rb

```diff
@@ -1,17 +1,25 @@
-require 'formula'
-
 class Kbtin < Formula
-  homepage 'http://kbtin.sourceforge.net'
-  url 'https://downloads.sourceforge.net/project/kbtin/kbtin/1.0.14/kbtin-1.0.14.tar.xz'
-  sha1 'b6e09ee7702fe52d48db53098c79fe243505cc0f'
-  revision 1
+  homepage "http://kbtin.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/kbtin/kbtin/1.0.15/kbtin-1.0.15.tar.xz"
+  mirror "https://mirrors.kernel.org/debian/pool/main/k/kbtin/kbtin_1.0.15.orig.tar.xz"
+  sha256 "57231dfd5dc0edcf71dcdafc16dc2e836c5d96e3b28927997bd2cc5061a313fa"
+
+  depends_on "gnutls"
 
-  depends_on 'gnutls'
+  if MacOS.version >= :mavericks
+    fails_with :clang do
+      cause "error: conflicting types for '__builtin___strlcpy_chk'"
+    end
+  end
```

Comment:
Can you link it in here too? Ta. I think we might wait for movement or an upstream patch before rolling out this version.